paddle/framework/tensor.h

```diff
@@ -164,6 +170,80 @@ class Tensor {
   size_t offset_;
 };
 
+/*
```

Comment:
I don't think sharing the same tensor header is a good idea.

Maybe we could extract LODTensor into `lod_tensor.h`.